python/perforce.py

```diff
@@ -228,6 +228,8 @@ def _write_interrupted(self):
             self.historical_interruption = True
         with open(self.interrupted_flag, 'w') as outfile:
             outfile.write("dirty")
+        with open(os.path.join(self.root, '.p4ignore'), 'w') as ignore_file:
```

Comment:
Or maybe the tombstone could be written to a known location outside of the actual checkout dir if needed?

Reply:
Yeah that's a good point, think I'll open it in append mode. We can't make assumptions about directory structure (within repo or without) since this is technically a public plugin. Our repo doesn't have a top level p4ignore, regardless `'a'` should cover us here.